net/tunneldigger/files/tunneldigger.init

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh /etc/rc.common
+
+. $IPKG_INSTROOT/lib/functions/network.sh
```

Comment:
It would be much nicer to make use of `procd` instead of the lengthy script below